docs/Researcher/workloads/inference-overview.md

```diff
@@ -50,34 +51,36 @@ This has the benefit of conserving resources at the risk of a delay from "cold s
 
-Once the new revision is created completely (according to the desired spec) and up and running, the entire traffic of requests is navigated to the new revision, and the original workload is terminated. Then the update process is considered complete.
+Once the new revision is created completely and up and running, the entire traffic of requests is navigated to the new revision, and the original workload is terminated. Then the update process is considered complete.
 
 It is important to note that:
 
-* To finish the inference workload update successfully, the project must have sufficient free GPU quota in favor of the update.  
-  For example:  
-    * Before the update: 3 replicas A running inference workload with 3 replicas (let's assume that each replica is equal to 1 GPU). This means the project is already using 3 GPUs of its quota. For the sake of simplicity, we will refer to this revision as revision #1.
+* To finish the inference workload update successfully, the project must have sufficient free GPU quota in favor of the update. For example:  
 
-    * The update: 8 replicas This means, to complete the update, an additional 8 GPUs of free quota is needed. Only when the update is complete, the 3 GPUs used for the 1st revision are reclaimed.
+    * The existing workload uses 3 replicas: A running inference workload with 3 replicas, assuming that each replica is equal to 1 GPU, means the project is already using 3 GPUs of its quota. For the sake of simplicity, we will refer to this revision as revision #1.
 
-* The Workload grid in the user interface always displays the configuration of the desired specification (the latest submitted update). The status of the workload still represents the service status. For example, per the example described in point 1, during the update, the status of the workload is still “running” as the service is still being provided to the consumers (using revision #1).
+    * The workload is updated to use 8 replicas: This means, to complete the update, an additional 8 GPUs of free quota is needed. Only when the update is complete, the 3 GPUs used for the initial running inference workload are reclaimed.
+
+* In the UI, the **Workloads table** displays the configuration of the latest submitted update. For example, if you change the number of replicas, the **running/requested pods** column will display the updated replicas requested. 
+The **status** of the workload, however, continues to reflect the operational state of the service as it was originally requested. For instance, during an update, the workload status remains "Running" if the service is still being delivered to consumers. Hovering over the workload's **status** in the grid will display the phase message for the update, offering additional insights into its update state.
```

Comment:
operational state of the service as it was originally requested --> operational state of the service the workload expose, as it was originally requested